src/main/java/teammates/logic/core/EmailGenerator.java

```diff
@@ -93,6 +123,37 @@
         return emails;
     }
     
+    private EmailWrapper generateFeedbackSessionEmailBaseForInstructorSubmissionConfirmation(
+            CourseAttributes course, FeedbackSessionAttributes session, InstructorAttributes instructor,
+            String template, String subject) {
+        
+        String submitUrl = Config.getAppUrl(Const.ActionURIs.INSTRUCTOR_FEEDBACK_SUBMISSION_EDIT_PAGE)
+                                 .withCourseId(course.getId())
+                                 .withSessionName(session.getFeedbackSessionName())
+                                 .toAbsoluteString();
+        
+        String reportUrl = Config.getAppUrl(Const.ActionURIs.INSTRUCTOR_FEEDBACK_RESULTS_PAGE)
+                                 .withCourseId(course.getId())
+                                 .withSessionName(session.getFeedbackSessionName())
+                                 .toAbsoluteString();
+        
+        String emailBody = Templates.populateTemplate(template,
+                "${userName}", instructor.name,
+                "${courseName}", course.getName(),
+                "${courseId}", course.getId(),
+                "${feedbackSessionName}", session.getFeedbackSessionName(),
+                "${deadline}", TimeHelper.formatTime12H(session.getEndTime()),
+                "${instructorFragment}", "",
+                "${submitUrl}", submitUrl,
+                "${reportUrl}", reportUrl,
+                "${supportEmail}", Config.SUPPORT_EMAIL);
+        
+        EmailWrapper email = getEmptyEmailAddressedToEmail(instructor.email);
+        email.setSubject(String.format(subject, course.getName(), session.getFeedbackSessionName()));
```

Comment:
If the subject is the same from all its call sites, just make it a default here. Saves us a function parameter.